.github/workflows/preview-fern-docs.yml

```diff
@@ -0,0 +1,61 @@
+name: Generate Fern Docs Preview
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  preview:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+
+    steps:
+      - name: Checkout aa-sdk
+        uses: actions/checkout@v4
+
+      - name: Download Alchemy Docs Repo
+        run: |
+          rm -rf docs-site && \
+          curl -L -f -H "Authorization: token ${{ secrets.DOCS_GITHUB_TOKEN }}" \
```

Comment:
This is a classic GitHub Token I supplied to allow us to access the currently private [docs repo](https://github.com/alchemyplatform/docs). We should remove it once the docs are made public.